lib/admin-page.php

```diff
@@ -617,10 +621,10 @@ function classicpress_check_can_migrate() {
 	}
 	echo "<p>\n";
 	printf( __(
-		/* translators: minimum supported PHP version */
-		'ClassicPress supports PHP versions <strong>%1$s</strong> and <strong>newer</strong>.',
+		/* translators: minimum supported PHP version, maximum supported PHP version */
```

Comment:
I had the wrong syntax here:

```suggestion
		/* translators: 1: minimum supported PHP version, 2: maximum supported PHP version */
```